### ISSUE TITLE
Build our own social sharing components

### ISSUE THREAD

**Issue by andyshuxin** [initiumlab/initiumlab.com#112]
Let's build our own social sharing component.
- [ ] Facebook
- [ ] Twitter
- [ ] Sina Weibo
- [ ] WeChat

**Comment by andyshuxin** [comment]
Implemented with https://github.com/initiumlab/hexo-theme-next/commit/4301da482e9dc29886fa8b2775308ba38cf7ce83
@yyfjmb You could switch to that branch and work on style

**Comment by Kateliuxue** [comment]
@andyshuxin The sharing button in lightbox do not work. Got no response when clicking on them.
windows 8.1, chrome

**Comment by andyshuxin** [comment]
@Kateliuxue Should work now, except for Facebook due to URL issues. Please check.

**Comment by Kateliuxue** [comment]
@andyshuxin Indeed. There's a small problem that when clicking on Wechat sharing button the QR Code appears beneath the screen making me think no response occurs.

**Comment by andyshuxin** [comment]
@yyfjmb Can you work on the styles of the button?

**Comment by yyfjmb** [comment]
@andyshuxin Sure. I'm working on it.

**Comment by yyfjmb** [comment]
@andyshuxin finished with https://github.com/initiumlab/hexo-theme-next/commit/b0818c5e5adf5ee5f0b64a5320fbb1b276fbaaa1
